eslint.config.mjs

```diff
@@ -0,0 +1,10 @@
+import neostandard, { resolveIgnoresFromGitignore } from 'neostandard'
+
+export default neostandard({
+  ts: true,
+  files: ['{fdm-core,fdm-data,fdm-calculator}/src/**/*.{js,mjs,cjs,ts}', 'fdm-app/app/**/*.{js,mjs,cjs,ts}'],
+  ignores: resolveIgnoresFromGitignore(),
+  rules: {
+    camelcase: 'off'
```

Comment:
If I run eslint, I still get errors for camelcase while I thought this rule is disabled. @coderabbitai, is the configuration correct and how to fix it?